manifests/a/Amazon/Games/latest/Amazon.Games.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Amazon.Games
+PackageVersion: latest
```

Comment:
```suggestion
PackageVersion: 1.7.3024.2
```

Reply:
The link always point to the latest version of the app.
When Amazon release a new version, the same link will install a newer version.

Reply:
PackageVersion needs to match Control Panel because Windows Package Manager uses this information for `winget list` and `winget upgrade`.

Reply:
wingetbot will automatically update the version when there's a new release.

Reply:
Oh, good to know!

This information needs to match for the app to be uninstalled successfully?